tests/apps/bookstore/bookstore-se/src/main/java/io/helidon/tests/apps/bookstore/se/AuthorMapper.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.tests.apps.bookstore.se;
+
+import javax.json.Json;
+import javax.json.JsonArray;
+import javax.json.JsonArrayBuilder;
+import javax.json.JsonBuilderFactory;
+import javax.json.JsonObject;
+
+import io.helidon.tests.apps.bookstore.mp.Author;
+
+import java.util.ArrayList;
+import java.util.List;
+
+class AuthorMapper {
+
+    private static final String FIRST = "first";
+    private static final String LAST = "last";
+
+    /*
+     * Use JsonBuilderFactory (and not the Json factory) to create builders.
+     * This lets us cache the builder factory and improves performance dramatically.
+     */
+    private static final JsonBuilderFactory jsonFactory = Json.createBuilderFactory(null);
```

Comment:
Incorrect name - constants should be all capital `JSON_FACTORY`

Reply:
Will fix.